src/applications/edu-benefits/1995/pages/stemEligibility.js

```diff
@@ -0,0 +1,29 @@
+import StemEligibilityView from '../containers/StemEligibilityView';
+import _ from 'lodash';
+
+const determineEligibilityIsNotChecked = formData =>
+  _.get(formData, 'determineEligibility') === undefined;
```

Comment:
I don't see why lodash needs to be used here:

```suggestion
  formData.determineEligibility === undefined;
```

Or this might even work:

```js
const determineEligibilityIsNotChecked = formData => !!formData.determineEligibility;
```